nanovdb/nanovdb/python/test/TestNanoVDB.py

```diff
@@ -1,4 +1,6 @@
-# /usr/bin/env python
+#!/usr/bin/env python
```

Comment:
@swahtz , @matthewdcong : is the first line necessary for this python script? Can we get rid of it?

Reply:
It's necessary if someone wants to run it directly `./TestnanoVDB.py`.
Also minor nit, it might be good to use `python3` instead of just `python`

Reply:
Got it. Thanks for the clarification @dgovil.